### ISSUE TITLE
login창 관련 질문

### ISSUE THREAD

**Issue by lumink** [phe14/demogracy#1]
프로그램이 시작됐을때 맨 처음 로그인부터 실행되도록 한 후
폰북과 스케줄 기능이 있는 창이 나타나야 하나요?
그렇다면 로그인을 위한 창을 만들어야 하는 건가요?

아니면 폰북과 스케줄 기능이 있는 창에 로그인 기능도 존재해야 하나요?

**Comment by phe14** [comment]
로그인 창을 따로 만들어서 로그인을 했을 경우에만 폰북과 스케줄이 있는 창으로 넘어가야 합니다. ^^

**Comment by democracyshr** [comment]
네, 메인 화면에 폰북과 스케줄을 한꺼번에 볼 수 있게 할 것이기 때문에, 로그인을 해야만 메인 화면을 볼 수 있도록 제한해야 합니다. 로그인을 위한 창을 따로 만들어야 해요.
